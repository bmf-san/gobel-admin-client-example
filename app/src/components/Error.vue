<template>
  <div class="error" v-if="error">
    <p>{{ error }}</p>
  </div>
</template>

<script>
export default {
  name: "Error",
  props: {
    error: String
  }
};
</script>

<style scoped>
.error p {
  color: red;
}
</style>
