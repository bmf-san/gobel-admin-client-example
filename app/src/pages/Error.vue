<template>
  <!-- TODO: update later -->
  <p>Error</p>
</template>

<script>
import { defineComponent } from "vue";
export default defineComponent({
  name: "Error"
});
</script>
