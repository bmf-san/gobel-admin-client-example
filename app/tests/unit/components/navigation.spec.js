import { shallowMount } from "@vue/test-utils";
import Error from "@/components/Error.vue";

describe("Error.vue", () => {
  it("init", () => {
    const $route = {
      path: "/some/path"
    };
    const wrapper = shallowMount(Error, {
      mock: {
        $route
      }
    });
    expect(wrapper.isVueInstance).toBeTruthy();
  });

  it("matched snapshot", () => {
    const $route = {
      path: "/some/path"
    };
    const wrapper = shallowMount(Error, {
      mock: {
        $route
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
