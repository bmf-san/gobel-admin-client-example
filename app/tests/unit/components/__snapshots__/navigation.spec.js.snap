// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error.vue matched snapshot 1`] = `""`;
